readme.txt

```diff
@@ -83,7 +83,7 @@ As of the WooCommerce 3.5 release, WooCommerce Services no longer provides shipp
 
 == Changelog ==
 
-= 1.24.1 - 2020-xx-xx =
```

Comment:
Do we usually also change this?

Reply:
Yeah, it's already changed in the latest `master` though. Maybe your branch needs to be rebased on latest `master`?